### DISCUSSION TITLE
fastq-dump -X 1 -Z SRR1553591 Error

### DISCUSSION THREAD

**Discussion by ShakySion** [biostars/biostar-handbook-issues#209] [Q&A] [answered]
Hello,

Per title, final doctor.py check reads OK for all except fastq-dump -X 1 -Z SRR1553591:

![image](https://user-images.githubusercontent.com/105201434/167575984-8aec69f9-6b63-4963-b996-ecd9e7d12222.png)

Followed the step-by-step installation instructions and things went relatively smoothly up to this point.  When printing the command directly, I get this:

![image](https://user-images.githubusercontent.com/105201434/167576413-c18aa572-3d93-45c6-8ab9-52c15e7f7fd7.png)

So I assumed that it was similar in nature to 

* <https://github.com/biostars/biostar-handbook/discussions/200>

Though since I'm an absolute newbie to this, when it gets to the point of adding the sra's bin to the path in .bashrc , I've got no clue how to do that.

So I guess I'm looking for two things here, 1: is the problem the same in nature/solution, and 2. if it is how can I fix it in a step-by-step process?

Thanks for your help!

**Comment by ialbert** [comment, marked as answer]
The commands to install the official version of the sratools would be:

```
# Go to bin directory
cd ~/bin

# Download the software
wget http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-ubuntu64.tar.gz

# Unpack the software 
tar -zxvf sratoolkit.current-ubuntu64.tar.gz

# Add the new directory to the path
echo 'export PATH=~/bin/sratoolkit.2.11.2-ubuntu64/bin:$PATH' >> ~/.bashrc

```
restart the terminal and check wich version of fastq-dump is found

    which fastq-dump

**Reply by ShakySion** [reply]
Huzzah! Thank you for your help! Worked like a charm!

**Reply by d2mcbrid** [reply]
Just a note on this. I encountered a similar issue (Linux subsystem on Windows 10). I did the step-by-step installation, reached this point, but after I performed the above, I wasn't able to use the conda command from my terminal. Long story short, when adding the new directory to the path, I had to use the ">>" command, not the ">" command to avoid overwriting the existing path. Hope this saves headaches for someone in the future.

**Reply by ialbert** [reply]
Yes, correct, I apologize for the typo, the `>>` is the append. I have now corrected the above

**Comment by ialbert** [comment]
There is an automated script now that installs sratools separately, which may be merged into the main install:

https://github.com/biostars/biostar-handbook/discussions/200#discussioncomment-3310589
